src/scenes/Arbitration/store.js

```diff
@@ -69,21 +69,23 @@ export default class ArbitrationStore {
   }
 
   async fetch(limit = this.limit, skip = this.skip) {
-    // if (this.hasMore) {
-    //   const orderBy = { field: 'endTime', direction: this.app.sortBy };
-    //   const excludeResultSetterAddress = this.app.wallet.addresses.map(({ address }) => address);
-    //   const filters = [
-    //     { token: Token.BOT, status: OracleStatus.VOTING, language: this.app.ui.locale },
-    //     { token: Token.QTUM,
-    //       status: OracleStatus.WAIT_RESULT,
-    //       excludeResultSetterAddress,
-    //       language: this.app.ui.locale,
-    //     },
-    //   ];
-    //   const { oracles, pageInfo: { hasNextPage } } = await queryAllOracles(this.app, filters, orderBy, limit, skip);
-    //   this.hasMore = hasNextPage;
-    //   return _.orderBy(oracles, ['endTime'], this.app.sortBy.toLowerCase());
-    // }
+    if (this.hasMore) {
+      const orderBy = { field: 'endTime', direction: this.app.sortBy };
+      const filters = [
+        { status: EVENT_STATUS.ARBITRATION, language: this.app.ui.locale },
+        { status: EVENT_STATUS.ORACLE_RESULT_SETTING,
+          language: this.app.ui.locale,
+        },
+      ];
+
+      const { account } = this.app.naka;
+      if (!account) await window.ethereum.enable();
+
+      const res = await events({ filters, orderBy, limit, skip, pendingTxsAddress: account });
+      if (res.pageInfo) this.hasMore = res.pageInfo.hasNextPage;
+      else this.hasMore = false;
+      return _.orderBy(res.items, ['endTime'], this.app.sortBy.toLowerCase());
```

Comment:
please import only what you need from lodash.

Reply:
change `endTime` -> `arbitrationEndTime`

Reply:
nit: change using `this.app.sortBy.toLowerCase()` to `constants.SortBy.DESCENDING.toLowerCase()`. not sure why we have a global sort by set in the app store?